plugin/xcode.vim

```diff
@@ -94,7 +99,13 @@ function! s:open(path)
 endfunction
 
 function! s:switch(target)
-  execute '!sudo xcode-select -s' . s:cli_args(a:target)
+  if has('nvim')
```

Comment:
That would actually  (slightly) change the current behavior. Right now we're using `!` to perform this command regardless of if they have a custom command set up or not. I'm not sure if that's the best thing to do long term, but I wanted to make that decision independently of this one.